src/components/Carousel/Carousel.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Meta, Story } from '@storybook/react'
+import Carousel from './index'
+import type { CarouselProps } from './index'
+import { IMAGE } from '@constants'
+
+export default {
+  argTypes: {},
+  component: Carousel,
+  title: 'Component/Carousel'
+} as Meta<CarouselProps>
+
+const Template: Story<CarouselProps> = args => <Carousel {...args} />
+
+export const isArrow = Template.bind({})
+isArrow.args = {
+  images: [
+    { id: 1, url: `${IMAGE.CATEGORY_CAR}` },
+    { id: 2, url: `${IMAGE.CATEGORY_APPLIANCE}` },
+    { id: 3, url: `${IMAGE.CATEGORY_APPLIANCE_PC}` }
+  ],
+  isArrow: true,
+  name: 'products',
+  size: 687
+}
+export const NoArrow = Template.bind({})
+
+NoArrow.args = {
+  images: [
+    { id: 1, url: `${IMAGE.CATEGORY_CAR}` },
+    { id: 2, url: `${IMAGE.CATEGORY_APPLIANCE}` },
+    { id: 3, url: `${IMAGE.CATEGORY_APPLIANCE_PC}` }
+  ],
+  isArrow: false,
+  name: 'products',
+  size: 687
+}
```

Comment:
주영 ! 내가 생각했던 거는 이렇게 변수명으로 지정해서 중복을 줄이는 느낌이었더 ㅎㅎ

```suggestion
const images =  [
    { id: 1, url: `${IMAGE.CATEGORY_CAR}` },
    { id: 2, url: `${IMAGE.CATEGORY_APPLIANCE}` },
    { id: 3, url: `${IMAGE.CATEGORY_APPLIANCE_PC}` }
  ]

export const isArrow = Template.bind({})
isArrow.args = {
  images,
  isArrow: true,
  name: 'products',
  size: 687
}
export const NoArrow = Template.bind({})

NoArrow.args = {
  images,
  isArrow: false,
  name: 'products',
  size: 687
}
```